static/js/timerender.ts

```diff
@@ -294,6 +294,8 @@ export function get_timestamp_for_flatpickr(timestring: string): Date {
             timestamp = new Date();
         }
     }
+    // set timestamp minutes to 0
+    timestamp.setMinutes(0, 0);
```

Comment:
Can you explain why this change needs to happen in two different files? 

In general, I'd really appreciate your pull requests having explanations of why the changes are correct / what investigation you did in addition to just a screencast; it's very hard to review something efficiently without those.

Reply:
@timabbott , Updated the PR with research details.